export function recordException(error: any, name: any): any;
